include/samurai/field.hpp

```diff
@@ -57,6 +57,7 @@ namespace samurai
             using interval_t = typename mesh_t::interval_t;
             using cell_t = Cell<typename interval_t::coord_index_t, dim>;
             using data_type = xt::xtensor<value_t, 1>;
+            using point_value_type = value_t;
```

Comment:
I would prefer that the Field class not be modified until we have a better implementation of the boundary conditions.
This type can be deduced directly in the BoundaryCondition class or using another class such as

```
template <class value_t, std::size_t size>
struct point_value_type
{
    using type = xt::xtensor_fixed<value_t, xt::xshape<size>>;
};

template <class value_t>
struct point_value_type<value_t, 1>
{
    using type = value_t;
};
```